src/utils/encoders.ts

```diff
@@ -64,6 +64,29 @@ export const encodeOrders = ([order0, order1]: [DecodedOrder, DecodedOrder]): [
   return [encodeOrder(order0), encodeOrder(order1)];
 };
 
+export const isOrderEncodable = (order: DecodedOrder) => {
+  try {
+    encodeOrder(order);
+    return true;
+  } catch (e) {
+    return false;
+  }
+};
+
+/**
+ * Checks if the rates are equal after scaling them
+ * @param {string} x - the first rate
+ * @param {string} y - the second rate
+ * @returns {boolean} - true if the rates are equal after scaling, false otherwise
+ */
+export const scaleRatesAndCheckIsEqual = (x: string, y: string) => {
```

Comment:
I'd name this one `scaledRatesAreEqual` or `areScaledRatesEqual`

Reply:
This was actually my first choice - but it felt to me that I should communicate better that the rates are first scaled and then compared. But I'll take your suggestion.